src/styles.scss

```diff
@@ -161,6 +161,19 @@ html {
                 }
             }
         }
+        .pagination {
+            display: inline-block;
+
+            a {
+                color: black;
```

Comment:
Should remove `color: black;`
Makes it looks like this ->

![image](https://user-images.githubusercontent.com/20019194/210079237-ac02b8df-354a-4f0f-a3b2-775369dced33.png)

Reply:
Whoops, didn't notice that because im using a chrome extension to change to dark mode all the webs. Thanks!

Reply:
> Should remove `color: black;` Makes it look like this ->
> 
> ![image](https://user-images.githubusercontent.com/20019194/210079237-ac02b8df-354a-4f0f-a3b2-775369dced33.png)

Fixed